config/clusteringress-istio.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
```

Comment:
We've recently added this label to allow users to install with `kubectl apply -f config/ --selector     networking.knative.dev/ingress-provider!=istio`.  See https://github.com/knative/serving/blob/master/config/202-gateway.yaml#L9 

Can you please add the same label?  Thanks